benchmarker.py

```diff
@@ -356,36 +350,42 @@ def plot_ups_consistency(
     plt.close()
 
 
-def plot_benchmark_results(outfile: str, folder: str, subfolder: str, errfile: str) -> None:
+def plot_benchmark_results(
+    data_table: list[list[int]],
+    titles: list[str],
+    maps: list[str],
+    folder: str,
+    subfolder: str,
+    errfile: list[list[int]],
+) -> None:
     """Generate plots of benchmark results."""
     # Create the output subfolder if it does not exist
     subfolder_path = os.path.join(folder, "graphs", subfolder)
     if not os.path.exists(subfolder_path):
         os.makedirs(subfolder_path)
 
-    for col in itertools.chain(range(1, 12), range(23, 33)):
+    for col in itertools.chain(range(1, 11), range(22, 32)):
         fig, ax = plt.subplots()
-        maps = column(outfile, 0)[1:]
-        update = column(outfile, col)[1:]
+        update = column(data_table, col)
         hbars = ax.barh(maps, update)
         ax.bar_label(
             hbars,
-            labels=[f"{x:.3f}" for x in column(outfile, col)[1:]],
+            labels=[f"{x:.3f}" for x in column(data_table, col)],
             padding=3,
         )
         ax.margins(0.1, 0.05)
-        ax.set_title(outfile[0][col])
+        ax.set_title(titles[col])
         ax.set_xlabel("Mean frametime [ms/frame]")
         ax.set_ylabel("Map name")
         plt.tight_layout()
         # Use os.path.join to build the file path for the output image
-        out_path = os.path.join(subfolder_path, f"{outfile[0][col]}.png")
+        out_path = os.path.join(subfolder_path, f"{titles[col+1]}.png")
```

Comment:
```suggestion
        out_path = os.path.join(subfolder_path, f"{titles[col]}.png")
```
the index must match with -> **ax.set_title(titles[col])** (line 377)

Reply:
fixed in db25f66